src/query/ast/tests/it/testdata/statement-error.txt

```diff
@@ -361,7 +371,7 @@ error:
   --> SQL:1:9
   |
 1 | PRESIGN INVALID @my_stage/path/to/file
-  | ------- ^^^^^^^ expected `DOWNLOAD`, `AtString`, `UPLOAD`, or <QuotedString>
+  | ------- ^^^^^^^ expected `DOWNLOAD`, `AtString`, `UPLOAD`, `URL`, or <QuotedString>
```

Comment:
Can we only change for the external table uri location? Since snowflake doesn't have this command, so we want add the 'URL' to it align with create external stage.
Here looks we apply URL to all uri location, the syntax hint 'URL' broken the syntax with snowflake.